### DISCUSSION TITLE
Youtube video link does not open in the latest IBAX_Pitch_Pitch_Deck.pdf file. you need to fix the problem

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#2032] [Bug Submission]
Youtube video link does not open in the latest IBAX_Pitch_Pitch_Deck.pdf file. you need to fix the problem. thanks

![Screenshot_2022-08-27-23-29-28-66_697425e880d9c9b1bd93f36a116bd45d](https://user-images.githubusercontent.com/96292241/187047793-2c649a54-2165-44ad-abd4-9fd765b6a293.jpg)

@ibaxio @yolandadadada

**Comment by HalloKyle** [comment]
We checked every video link in pitchdeck and all of them are working fine. You might want to double check your internet connection?

**Comment by HalloKyle** [comment]
Again Thanks for your suggestion
